curriculum/challenges/english/08-coding-interview-prep/rosetta-code/kaprekar-numbers.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 5a23c84252665b21eecc7eca
+title: Kaprekar numbers
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+A positive integer is a <a href="https://en.wikipedia.org/wiki/Kaprekar number">Kaprekar number</a> if:
+<ul>
+<li>It is 1</li>
+<li>The decimal representation of its square may be split once into two parts consisting of positive integers which sum to the original number. </li>
+</ul>
+Note that a split resulting in a part consisting purely of 0s is not valid, as 0 is not considered positive.Example
+Kaprekar numbers:
+<ul>
+<li>$2223$ is a Kaprekar number, as $2223 * 2223 = 4941729$, $4941729$ may be split to $494$ and $1729$, and $494 + 1729 = 2223$</li>
```

Comment:
Rather than doing `$2223$`, if you do, `<code>2223</code>`, I think it'd be better. What do you think @scissorsneedfoodtoo ?

Reply:
@NitronR Just make this change, then we can review the PR